index.js

```diff
@@ -116,14 +119,18 @@ Object.keys(tabletUserAgents).forEach(function(k) {
    tabletRegex.push(new RegExp(tabletUserAgents[k], 'i'));
 });
 
-var userAgentCache = {};
-
+var userAgentCache = new LRU({max: 10000});
```

Comment:
I think would be better to make this configurable (just put it into a const on the top for now). And please make clear what it is (KB, MB?)